src/Qir/Tests/OpenSystemsSimulator/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+
+compile_from_qir(qir-test-opensimulator qir_test_opensimulator_target)
```

Comment:
This macro has been updated since this was worked on, so this folder won't build (and is indeed skipped intentionally for now, I see). Since it's not part of the build and not yet ready for integration, maybe it makes sense to tease out the QIR-opensim infrastructure for now and leave that for a later PR? Not a big deal while this lives in a feature branch, but not something I think we'd want in main, and merging to the feature branch as-is puts the burden on us to remember to re-review this part before merging the feature branch to main, when it might be eaiser to miss in a large diff.

Reply:
Would it perhaps make sense, then, to revert all changes to QIR runtime as part of this feature work? As per blocker on target packages in Python, QIR integration is likely deferred for now, and that keeps outdated code from landing in main by accident.

Reply:
Yeah, I do think that makes sense, especially now that you also filed the issue that helps track the QIR integration feedback you've already received.

Reply:
Sounds good, went on and reverted in f3c038258b54eec549169d1717f3dd8ec2411da6. If we need to, can always look at the parents for that commit. Thanks!